pkgs/by-name/li/libpqxx/package.nix

```diff
@@ -1,28 +1,51 @@
-{ lib, stdenv, fetchFromGitHub, postgresql, python3 }:
+{
+  fetchFromGitHub,
+  lib,
+  postgresql,
+  python3,
+  stdenv,
+}:
 
 stdenv.mkDerivation rec {
```

Comment:
```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Reply:
This is using a fixed point for no good reason. `rec` has better performance eval wise.